buildbuddy.yaml

```diff
@@ -10,6 +10,18 @@ actions:
           - "*"
     bazel_commands:
       - test //... --config=linux-workflows --config=race --test_tag_filters=-performance,-webdriver,-docker,-bare
+  - name: Test FOSS
+    container_image: ubuntu-20.04
+    triggers:
+      push:
+        branches:
+          - "master"
+      pull_request:
+        branches:
+          - "*"
+    bazel_commands:
+      - run --config=linux-workflows //tools:rm-enterprise
+      - build --config=linux-workflows //server
```

Comment:
to make sure the AC keys here are shared with the Test workflow, we need to add `--config=race` to both of these

Reply:
Good point. 👍 
Added.